spec/factories/volunteers.rb

```diff
@@ -7,6 +7,15 @@
     waiver_signed true
     waiver_signed_at Time.zone.now
 
+    factory :volunteer_with_region do
+      after(:create) do |v|
+        region = create(:region)
+        v.regions << region
+        v.assigned = true
+        v.save
+      end
+    end
```

Comment:
Hi @ericfreese 
For this particular test case this is being used to evaluate functionaliry when a `volunteer` without any `assignment` is trying to schedule an `absence`. It means not having an `assignment` is a precondition.
Since other tests are relying on the already existing, previously created factories, I didn't want to rename any previously created `volunteer` factories.
First I tried to rely on `volunteer` factory as it does not have `assignments` either, but that no `regions` assigned and its login fails with the `Your account has not been assigned to a region yet. A region administrator must do this before you can sign in.` error message.
This new factory is being called under `let(:volunteer_without_assignment)` within `volunteer_schedule_absence_spec.rb` It is already being called as `volunteer_without_assignment`, hoping that would clear up any possible confusion around the purpose of this factory.
Is there another `factory` I might have missed that already has volunteer with regions assigned without assignments?